src/LCM2ROSControl.cpp

```diff
@@ -41,7 +41,18 @@ namespace valkyrie_translator
           return false;
         }
         handler_ = std::shared_ptr<LCM2ROSControl_LCMHandler>(new LCM2ROSControl_LCMHandler(*this));
-        
+
+        // Check which joints we have been assigned to
+        // If we have joints assigned to just us, claim those, otherwise claim all
+        std::vector<std::string> joint_names_;
+        if (!controller_nh.getParam("joints", joint_names_))
+          ROS_INFO_STREAM("Could not get assigned list of joints, will resume to claim all");
```

Comment:
I don't like splitting the config between ROS params and lcm-config and director-json (and possibly also drake config).
this is the kind of decentralisation that I abhor in ROS.

But for now we will use this.